modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsStatsUserLocalServiceImpl.java

```diff
@@ -58,27 +60,23 @@
 	public List<BlogsStatsUser> getGroupsStatsUsers(
 		long companyId, long groupId, int start, int end) {
 
+		Predicate companyIdEquals = _companyIdAlias.eq(companyId);
```

Comment:
Can you inline this variable? Same with other `*IdEquals` variables.

Reply:
Hi @adolfopa I originally inlined the expressions, but then the source formatter complained about chaining of the eq() and and() methods.